File: Feitos em Java/2518.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);
        while(leitor.hasNext()){
            int degraus = leitor.nextInt();
            int h=0,c=0,l=0;
            h = leitor.nextInt();
            c = leitor.nextInt();
            l = leitor.nextInt();
            double compRampa=Math.sqrt(Math.pow(c,2)+Math.pow(h,2));
            compRampa*=degraus;
            double area=compRampa*l;
            System.out.printf("%.4f\n",area/10000);
        }
    }
}

File: Feitos em C/2518.c
#include <stdio.h>
#include <math.h>
int main() {
	int degraus;
	while(scanf("%d",&degraus)==1){
		int h,c,l;
		scanf("%d",&h);
		scanf("%d",&c);
		scanf("%d",&l);
		double compRampa=sqrt(pow(c,2)+pow(h,2));
		compRampa*=degraus;
		double area=compRampa*l;
		printf("%.4f\n",area/10000);
	}
	return 0;
}
